torch/_torch_docs.py

```diff
@@ -4395,7 +4396,8 @@
 
 Arguments:
     b (Tensor): RHS tensor.
-    A_LU (Tensor, IntTensor, fmt): LU-factorization of A from btrifact.
+    A_LU_data (Tensor): Pivoted LU factorization of A from btrifact.
```

Comment:
can you remove the `A_` prefix? They can be used as keywords so you should stick to the names from cwrap.